app/src/js/widgets/fields/fieldRepeater.js

```diff
@@ -246,10 +246,10 @@
 
             if (this._count >= this.options.maximum) {
                 this._ui.add.addClass('disabled');
-                this.element.find('.duplicate-button').addClass('disabled');
+                this._ui.add.prop('disabled', 'disabled');
             } else {
                 this._ui.add.removeClass('disabled');
-                this.element.find('.duplicate-button').removeClass('disabled');
+                this._ui.add.prop('disabled', false);
```

Comment:
and this line underneath here:

    this.element.find('.duplicate-button').prop('disabled', false);